<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphExplorer</class>
 <widget class="QWidget" name="GraphExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>918</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph Explorer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="graphNameLayout">
     <item>
      <widget class="QLabel" name="graphName">
       <property name="text">
        <string>Graph name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="graphVizLayout">
     <item>
      <widget class="QWidget" name="graphVizWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="graphImage">
          <property name="text">
           <string>Graph Image</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="nodesFilterProperty"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nodes</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="nodesFilterValue"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Property</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QTreeWidget" name="nodesTree">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QCheckBox" name="isGeometry">
       <property name="text">
        <string>Geometry</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="isActual">
       <property name="text">
        <string>Actual Graph</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0" colspan="4">
      <widget class="QTreeWidget" name="edgesTree">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Graph</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Edges</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QTreeWidget" name="graphTree">
       <property name="editTriggers">
        <set>QAbstractItemView::AllEditTriggers</set>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="3" column="0" colspan="5">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="3">
        <widget class="QLineEdit" name="edgesFilterValue"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Property</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edgesFilterProperty"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
